docs/site/markdown/index.md

```diff
@@ -45,5 +45,6 @@ This BeanUtils library is a utility library for managing Bean objects. The libra
 - #### Related articles
 
     * DZone: [How to Transform Any Type of Java Bean With BULL](https://dzone.com/articles/how-to-transform-any-type-of-java-bean-with-one-li)
-    * InfoQ: [How Expedia Is Getting Rid of Java Bean Transformers](https://www.infoq.com/articles/expedia-rid-of-bean-transformers/)
+    * InfoQ : [How Expedia Is Getting Rid of Java Bean Transformers](https://www.infoq.com/articles/expedia-rid-of-bean-transformers/)
+    * InfoQ Brazil: [Como a Expedia Está Se Livrando dos Transformadores de Java Beans](https://www.infoq.com/br/articles/expedia-rid-of-bean-transformers/)
```

Comment:
Instead of having two lines for the same article (even if it's in different languages), I would suggest leaving a single line with both links, it should look like as follows:

`- InfoQ: [How Expedia Is Getting Rid of Java Bean Transformers](https://www.infoq.com/articles/expedia-rid-of-bean-transformers/) [[EN](https://www.infoq.com/articles/expedia-rid-of-bean-transformers/)] [[BR](https://www.infoq.com/br/articles/expedia-rid-of-bean-transformers/)]`

the result will be:

- InfoQ: [How Expedia Is Getting Rid of Java Bean Transformers](https://www.infoq.com/articles/expedia-rid-of-bean-transformers/) [[EN](https://www.infoq.com/articles/expedia-rid-of-bean-transformers/)] [[BR](https://www.infoq.com/br/articles/expedia-rid-of-bean-transformers/)]

what do you think?

Reply:
Perfect! That's better and simple!